site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: Breached Password Detection
+description: Learn how to create a passwordless experience for your end users.
+---
+:page-liquid:
+
+== What Is Breached Password Detection
+
+Breached password detection consists of the following tasks:
+
+. Finding breached and compromised passwords.
+. Building a system to download, process and store these datasets.
+. Checking passwords to see if they've been compromised.
+. Taking action when a leaked credential is found.
+
+FusionAuth has taken care of steps 1, 2 and 3. All you need to do is to configure your breached password detection options.
```

Comment:
We imply they need to complete step 4 which is to take action. Should we re-phrase this to say 

> ... all you need to do is configure the desired action. 

Or something similar?